Examples/Algorithms/TruthTracking/ActsExamples/TruthTracking/ParticleSmearing.cpp

```diff
@@ -143,7 +143,7 @@ ActsExamples::ProcessCode ActsExamples::ParticleSmearing::execute(
 
             // theta contribution
             variance += varianceTheta *
-                        std::pow(params[Acts::eBoundQOverP] *
+                        std::pow(params[Acts::eBoundQOverP] /
```

Comment:
Here we are the user and create a `theta` that might be `0`:
```
const auto [newPhi, newTheta] = Acts::detail::normalizePhiTheta(
    phi + sigmaPhi * stdNormal(rng), theta + sigmaTheta * stdNormal(rng));
```

Reply:
This is outside my concern for this PR

Reply:
I started a new PR to fix this:
- https://github.com/acts-project/acts/pull/3687